apertium-eng-cat.eng-cat.dix

```diff
@@ -11,99 +11,116 @@
+    <sdef n="guio"  c="Hyphen"/> 
+    <sdef n="apos"  c="Apostrophe"/> 
+<!--Gender-->
+    <sdef n="f" c="	Feminine"/> 
+    <sdef n="m"  c="Masculine"/> 
+    <sdef n="mf"    c="Masculine or feminine"/>
+    <sdef n="nt"  c="Neuter"/>
+    <sdef n="GD"  c="Gender to be determined	"/> 
+<!--Count/Mass-->
+    <sdef n="unc"  c="Uncountable (mass)"/>
+<!--Animacy-->
+    <sdef n="nn" c="Inanimate"/> 
+    <sdef n="an" c="Animate or inanimate"/> 
+    <sdef n="aa" c="Animate"/> 
+<!--Adjectives-->
+<sdef n="pred"  c="Predicative"/> 
+<!--Pronoun Types-->
+    <sdef n="qnt"  c="Quantifier"/> 
+    <sdef n="enc" c="Enclitic"/> 
+    <sdef n="pro" c="Proclitic"/> 
+    <sdef n="tn" c="Tónico"/>
+    <sdef n="predet"  c="Predeterminer"/> 
+    <sdef n="acr"  c="Acronym"/> 
+    <sdef n="rel"  c="Relative"/>
+    <sdef n="ind"  c="Indefinite"/> 
+    <sdef n="itg"  c="Interrogative"/> 
+    <sdef n="det"  c="Determiner"/> 
+    <sdef n="dem"  c="Demonstrative"/> 
+    <sdef n="def"  c="Definite"/> 
+    <sdef n="detnt"  c="Neuter determiner"/> 
+    <sdef n="atn" c="Atónico"/> 
+    <sdef n="pos"  c="Possessive"/> 
+    <sdef n="ref"  c="Reflexive"/>
+    <sdef n="subj"  c="Subject"/> 
+    <sdef n="obj"  c="Object	"/> 
+    <sdef n="ord"  c="Ordinal"/>  
+<!--Separable Verbs-->
+    <sdef n="sep"  c="Separable verb"/> 
+<!--Proper Nouns-->
+    <sdef n="top"  c="Toponym"/> 
+    <sdef n="ant"  c="Anthroponym"/>
+    <sdef n="al"  c="Altres"/> 
+    <sdef n="cog" c="Cognomen"/>
+    <sdef n="org"  c="Organisation"/> 
+<!--Number-->
+    <sdef n="sg"    c="Singular"/> 
+    <sdef n="pl"  c="Plural"/> 
+    <sdef n="ND"  c="Number to be determined"/> 
+    <sdef n="sp"  c="Singular or plural	"/> 
+<!--case-->
+    <sdef n="gen"  c="Genitive"/> 
+<!--Tense and Mode-->
+    <sdef n="inf"  c="Infinitive"/> 
+    <sdef n="pres"  c="Present"/> 
+    <sdef n="ger"  c="Gerund"/> 
+    <sdef n="pp"  c="Past participle"/> 
+    <sdef n="pprs"  c="Present participle"/> 
+    <sdef n="pii"  c="Imperfect"/> 
+    <sdef n="ifi"  c="Past Indefinite"/> 
+    <sdef n="fti"  c="Future Indicative"/> 
+    <sdef n="cni"  c="Conditional"/> 
+    <sdef n="prs"  c="Present Subjunctive"/> 
+    <sdef n="pis"  c="Imperfect Subjunctive"/> 
+    <sdef n="imp" c="Imperative"/> 
+    <sdef n="past"  c="Past"/> 
+<!--Person-->
+    <sdef n="p1"  c="First Person"/> 
+    <sdef n="p2"  c="Second Person"/> 
+    <sdef n="p3"  c="Third Person"/> 
+<!--Derivations-->
+    <sdef n="subs"  c="Verbal Noun or Verbal Substantive"/> 
+<!--Adjectives-->
+    <sdef n="comp"  c="Comparative"/> 
+    <sdef n="sup"  c="Superlative"/> 
+    <sdef n="sint"  c="Synthetic"/> 
+<!--Others-->
+    <sdef n="web"  c="Links and Emails"/> 
+    <sdef n="percent"  c="Percentage"/> 
+    <sdef n="mon"  c="Money"/> 
+    <sdef n="maj"  c="majuscule"/> 
+    <sdef n="min"  c="Miniscule"/> 
+    <sdef n="file"  c="Filename"/>
+    <sdef n="adjant"  c=""/> 
     <sdef n="pron" /> 
```

Comment:
This tag was missing.
It is perfect now.

Reply:
Why did you add it under "Others"?

Reply:
I did so becuase 'pron' could not be found anywhere on the list of symbols page

Reply:
It means "pronoun"

Reply:
Should probably be moved to where it goes.

Reply:
I have moved it to POS category